static/templates/compose.hbs

```diff
@@ -46,7 +46,13 @@
         </div>
     </div>
     <div class="message_comp">
-        <div id="compose_banners"></div>
+        <div id="compose_banners">
+            <div id="upload_banner" class="hidden compose_banner info">
+                <div class="moving_bar"></div>
+                <p class="upload_msg"></p>
+                <a role="button" class="zulip-icon zulip-icon-close compose_banner_close_button"></a>
+            </div>
+        </div>
```

Comment:
This looks like a singleton -- how does it work if multiple files are being uploaded at once?

Reply:
Ah yeah that will be fixed by [this](https://github.com/zulip/zulip/pull/23486#discussion_r1046448383). I'll mark this PR as no longer a draft when it's ready for review.

Reply:
It's actually currently implemented as a singleton on main. That's good that we can fix that here.

Reply:
Okay so making it not a singleton is a bit of a doozy and should probably happen in a separate PR. Do you have a preference for that happening first or this redesign happening first?

Reply:
Aha! So actually if there are multiple files being uploaded at once, they get combined into a single progress bar and the bar only goes away once the upload is complete. I'm okay keeping it like that for now, since it's a lot simpler than trying to find a unique identifier for the files and keep track of the banners

Reply:
Yeah, I think a single progress bar for the bundle is probably fine in practice. There's probably a way to configure `uppy` to do something fancier if we need it, but that seems like a distraction from completing the migration to the new banner styling.